frontend/src/pages/servers/index.tsx

```diff
@@ -315,13 +319,14 @@ const Page = ({ initialData, errorMessage }: Props) => {
   }
 
   return (
-    <section className="measure-wide center ph2 pb2">
+    <section style={{ maxWidth: "50em", margin: "auto", padding: "1em 2em" }}>
```

Comment:
More idiomatic Chakra usage:

```suggestion
    <Box as="section" maxWidth="50em" margin="auto" padding="1em 2em">
```